### ISSUE TITLE
Unable to build the project using Ninja

### ISSUE THREAD

**Issue by Michcioperz** [sio2project/sio2jail#6]
`cmake -G Ninja -DCMAKE_BUILD_TYPE=Release ..` proceeds fine, but `ninja` afterwards fails.

An example error message:
```
ninja: error: 'seccomp_project-prefix/lib/libseccomp.a', needed by 'src/sio2jail', missing and no known rule to make it
```

**Comment by teqwve** [comment]
It looks like this: https://cmake.org/pipermail/cmake/2015-April/060234.html should fix it.

**Comment by teqwve** [comment]
Fixed in 9abc744801d122737c90c25ffe13229f89a3f299
